app/controllers/admin/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::DashboardController < Admin::BaseController
```

Comment:
Per Toni's rant, we shall ignore these specific hound complaints until instructed not to.